rust/main/chains/hyperlane-sealevel/src/priority_fee.rs

```diff
@@ -0,0 +1,225 @@
+#![allow(dead_code)]
```

Comment:
We can remove it if we add annotations directly to fields which are not used:

```
#[derive(Debug, Deserialize)]
#[serde(rename_all = "camelCase")]
struct JsonRpcResult<T> {
    #[allow(dead_code)]
    jsonrpc: String,
    #[allow(dead_code)]
    id: String,
    result: T,
}
```

Reply:
ty, forgot to remove this